app/src/main/java/com/eventyay/organizer/data/db/configuration/OrgaDatabase.java

```diff
@@ -310,4 +312,21 @@ public void onPreMigrate() {
         }
     }
 
+    @Migration(version = 19, database = OrgaDatabase.class)
+    public static class MigrationTo19 extends BaseMigration {
+
+        @Override
+        public void migrate(@NonNull DatabaseWrapper databaseWrapper) {
+            Timber.d("Running migration for DB version 19");
+
+            Class<?>[] recreated = new Class[] {Role.class, RoleInvite.class};
+
+            for (Class<?> recreate: recreated) {
```

Comment:
Why `?`?

Reply:
To create an array of classes.

Reply:
https://docs.oracle.com/javase/7/docs/api/java/lang/Class.html